packages/security/src/components/GettingStartedLayout/docs/Troubleshooting/index.mdx

```diff
@@ -0,0 +1,28 @@
+import github from './images/github.png';
+import slack from './images/slack.png';
+
+### Troubleshooting
+
+Visit our [GitHub](https://github.com/carbon-design-system/ibm-security) to view
```

Comment:
this link should point to https://github.com/carbon-design-system/ibm-cloud-cognitive/tree/main/packages/security